Update rare-ordinal-directory.json

subterranean1 opened the pull request:
I fixed (3) broken links and added 10 new Series 4 cards.

subterranean1 committed:
Update rare-ordinal-directory.json

I fixed (3) broken links and added 10 new Series 4 cards.

subterranean1 committed:
Update rare-ordinal-directory.json

Added new series 4 cards

subterranean1 committed:
Update rare-ordinal-directory.json

Completed Series 4

subterranean1 committed:
Update rare-ordinal-directory.json

current up to series 5, card 25

btroia commented:
> I fixed (3) broken links and added 10 new Series 4 cards.

Hi @subterranean1! Looks like there is some extra space or special characters in your json file, and it doesn't validate.

Please, _always_ paste your json code into a [json validator](https://jsonlint.com/) to make sure there are no errors 👍

<br />

<img width="744" alt="Screen Shot 2023-04-04 at 2 00 08 PM" src="https://user-images.githubusercontent.com/368747/229894289-adbb68ef-5388-4082-b2e4-b5047d942e54.png">

subterranean1 committed:
Update rare-ordinal-directory.json

Fixed extra spaces

subterranean1 commented:
Hello,

I have corrected the error in the Rare Ordinals file. Can you please also
see my PR for Bitcoin Stamps? I have verified that it is a valid file. If
you get it added to the app, I will update it to the current stamp number.

Thank you

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

On Tue, Apr 4, 2023 at 12:03 PM Bob Troia ***@***.***> wrote:

> I fixed (3) broken links and added 10 new Series 4 cards.
>
> Hi! Looks like there is some extra space or special characters in your
> json file, and it doesn't validate.
>
> Please, *always* paste your json code into a json validator
> <https://jsonlint.com/> to make sure there are no errors 👍
>
> [image: Screen Shot 2023-04-04 at 2 00 08 PM]
> <https://user-images.githubusercontent.com/368747/229894289-adbb68ef-5388-4082-b2e4-b5047d942e54.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/AwesomeLabs/token-art/pull/7#issuecomment-1496459353>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ASKVXVTZ7E6WLE2RZ2EEP3LW7RWBPANCNFSM6AAAAAAVKGQDGE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
